lib/components/Navbar.js

```diff
@@ -46,52 +15,81 @@ class Navbar extends Component {
     store: PropTypes.object.isRequired
   };
 
-  render() {
-    // const styles = require('./Navbar.scss');
-    const {user, logout} = this.props;
+  componentDidMount() {
 
-    let iconMenuItems = [
-      {payload: '1', text: 'Profile'},
-      {payload: '2', text: 'Map'}
-    ];
+    const menu = document.getElementById('menu'),
+      WINDOW_CHANGE_EVENT = ('onorientationchange' in window) ? 'orientationchange' : 'resize';
 
-    const signInOutBtnStyles = {
-      backgroundColor: 'transparent',
-      signInOutBtnStyles
+    function toggleHorizontal() {
```

Comment:
Just thought it was weird that they were inside that function instead of being defined at the top of the file.